File: to-do app/to-do-go/src/router/routes/routesUser.go
package routes

import (
	"net/http"

	"renan.com/todo/src/controller"
)

var userRoutes = []Route{
	{
		URI:       "/api/user/profile",
		Method:    http.MethodPut,
		Function:  controller.UpdateUserProfile,
		NeedsAuth: true,
	},
	{
		URI:       "/api/user/password",
		Method:    http.MethodPut,
		Function:  controller.UpdateUserPassword,
		NeedsAuth: true,
	},
}

File: to-do-app/to-do-node/src/controller/routes/routesUser.ts
import express from 'express';
import userController from '../userController';
import mid from '../middleware/middleware'

const router = express.Router();

router.put('/api/user/profile', mid.validateAuth, userController.updateUserProfile)
router.put('/api/user/password', mid.validateAuth, userController.updateUserPassword)

export default { router };